### PR TITLE
Target Jellyfin 10.7

### PR DESCRIPTION
**Pull request by oddstr13**
- `IHttpClient` -> `IHttpClientFactory`

### CODE REVIEW

**Comment by oddstr13** [comment]
CI fails due to missing .net 5

**Comment by oddstr13** [comment]
Current CI faulure is due to building against 10.6

File: Jellyfin.Plugin.TvMaze/Providers/TvMazeEpisodeImageProvider.cs [hunk 1]
```diff
@@ -106,13 +107,10 @@ public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, Cancell
         }
 
         /// <inheritdoc />
-        public Task<HttpResponseInfo> GetImageResponse(string url, CancellationToken cancellationToken)
+        public async Task<HttpResponseMessage> GetImageResponse(string url, CancellationToken cancellationToken)
```
Comment (crobibero) [inline Jellyfin.Plugin.TvMaze/Providers/TvMazeEpisodeImageProvider.cs:110]:
```suggestion
        public Task<HttpResponseMessage> GetImageResponse(string url, CancellationToken cancellationToken)
```

File: Jellyfin.Plugin.TvMaze/Providers/TvMazeEpisodeImageProvider.cs [hunk 2]
```diff
@@ -106,13 +107,10 @@ public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, Cancell
...
         {
-            return _httpClient.GetResponse(new HttpRequestOptions
-            {
-                CancellationToken = cancellationToken,
-                Url = url
-            });
+            var httpClient = _httpClientFactory.CreateClient(NamedClient.Default);
+            return await httpClient.GetAsync(new Uri(url), cancellationToken).ConfigureAwait(false);
```
Comment (crobibero) [inline Jellyfin.Plugin.TvMaze/Providers/TvMazeEpisodeImageProvider.cs:113]:
```suggestion
            return _httpClientFactory.CreateClient(NamedClient.Default).GetAsync(new Uri(url), cancellationToken);
```

**Review by crobibero — COMMENTED** [review]
Can simplify all `GetImageResponse` requests

**Review by crobibero — APPROVED** [review]

**Pull request merged**